tests/unit/scm/git.py

```diff
@@ -0,0 +1,14 @@
+from tests.basic_env import TestDvc
+
+
+class TestGit(TestDvc):
+    def test_belongs_to_scm_true_on_gitignore(self):
+        self.assertTrue(self.dvc.scm.belongs_to_scm("path/to/.gitignore"))
```

Comment:
Please don't hardcode `/`. Use os.path.join instead. It will test it on windows then as well.

Reply:
I keep forgetting about this one, sorry.